src/dump/dump.c

```diff
@@ -109,9 +109,6 @@ void dump_write_score(FILE *f, gregorio_score *score)
     if (score->score_copyright) {
         fprintf(f, "   score_copyright           %s\n", score->score_copyright);
     }
-    if (score->language) {
-        fprintf(f, "   language                  %s\n", score->language);
-    }
```

Comment:
All headers (except the `def-m*` headers) are actually now stored in `gregorio_score.headers`.  The section you are seeing here dumps values that are additionally stored in the `gregorio_score` structure, and there's no need to store language in that structure, so I removed it from the structure and therefore from the dump.  If you take a look at https://github.com/henryso/gregorio-test/blob/fix-756/tests/gabc-dump/test_all_headers.dump you'll see that language is still dumped in the "Headers" section.